vnext/ReactUWP/Views/ControlViewManager.cpp

```diff
@@ -37,6 +37,7 @@ void ControlViewManager::TransferProperties(
   TransferProperty(oldView, newView, winrt::Control::PaddingProperty());
   TransferProperty(oldView, newView, winrt::Control::ForegroundProperty());
   TransferProperty(oldView, newView, winrt::Control::TabIndexProperty());
+  __super::TransferProperties(oldView, newView);
```

Comment:
Nit: I don't know which one is more reasonable, it's better to be consistent with `___super::` or `Super::`
I see the next function is using `Super::UpdateProperties`

Is there any reason that super:: call is not the first line of function?

Reply:
I will change to follow the convention here:, Super is defined in ControlViewManager.h: using Super = FrameworkElementViewManager;

---
In reply to: [322875317](https://github.com/microsoft/react-native-windows/pull/3108#discussion_r322875317) [](ancestors = 322875317)